index.js

```diff
@@ -12,12 +12,24 @@
 
 'use strict';
 
+const Action = require("./lib/action");
+const Asset = require("./lib/asset");
+const Prepare = require("./lib/prepare");
+const Rendition = require("./lib/rendition");
+const Storage = require("./lib/storage");
+const Utils = require("./lib/utils");
 const Transformer = require("./lib/transformer");
 const {Plan} = require("./lib/plan");
 const Engine = require("./lib/engine");
 const Manifest = require("./lib/manifest");
 
 module.exports = {
+    Action,
+    Asset,
+    Prepare,
+    Rendition,
+    Storage,
+    Utils,
```

Comment:
do we also want to export the errors and log utils from asset-compute-commons like we do in the sdk? https://github.com/adobe/asset-compute-sdk/blob/master/index.js#L16-L42